diesel_tests/tests/schema/mod.rs

```diff
@@ -90,6 +90,44 @@ impl Comment {
     }
 }
 
+#[derive(
+    PartialEq,
+    Eq,
+    Debug,
+    Clone,
+    Queryable,
+    AsChangeset,
+    Insertable,
+    Identifiable,
+    Associations,
+    Selectable,
+)]
+#[diesel(belongs_to(Post), table_name = comments)]
+pub struct CommentRc {
+    pub id: i32,
+    pub post_id: i32,
+    pub text: std::rc::Rc<str>,
```

Comment:
I would expect that this causes issues for loading values as we don't have a `ToSql<DB, Text> for str` impl yet. That said it seems reasonable to support this as well, but I would like to keep that off this PR to not expand it's scope any further. So just go with `Rc<String>` for now.